pcdsdevices/lic_2d_tmo.py

```diff
@@ -28,6 +39,7 @@ class TMOLaserInCouplingTwoDimension(BaseInterface, GroupDevice, LightpathMixin)
     tab_component_names = True
 
     # LaserCoupling LI2K4 x and Y
+    laser_incoupling = Cpt(LaserCouplingStates, 'LI2K4:IP1:STATE', add_prefix=(), kind='normal')
```

Comment:
I think this is reasonable, but I want to double-check the prefixes and how the class is launched, and the filesystem is too broken right now to continue.